src/google/cloud/happybase/connection.py

```diff
@@ -273,7 +273,7 @@ def tables(self):
 
         return table_names
 
-    def create_table(self, name, families):
+    def create_table(self, name, families, initial_split_keys=[]):
```

Comment:
You can't change this interface.  Please revert the use of `initial_split_keys`.  If you need split keys, use the underlying google-cloud-python connection instead.